source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -122,12 +188,26 @@ class ZooKeeperFilterConfig {
   const Stats::StatName unknown_scheme_rq_;
   const Stats::StatName unknown_opcode_latency_;
 
+  ErrorBudgetResponseType errorBudgetDecision(const OpCodes opcode,
+                                              const std::chrono::milliseconds& latency);
+
 private:
-  void initOpCode(OpCodes opcode, Stats::Counter& counter, absl::string_view name);
+  void initOpCode(OpCodes opcode, Stats::Counter& resp_counter, Stats::Counter& resp_fast_counter,
+                  Stats::Counter& resp_slow_counter, absl::string_view name);
 
   ZooKeeperProxyStats generateStats(const std::string& prefix, Stats::Scope& scope) {
     return ZooKeeperProxyStats{ALL_ZOOKEEPER_PROXY_STATS(POOL_COUNTER_PREFIX(scope, prefix))};
   }
+
+  int32_t getOpCodeIndex(LatencyThreshold_Opcode opcode);
+  absl::flat_hash_map<int32_t, std::chrono::milliseconds>
+  parseLatencyThresholds(LatencyThresholdList& latency_thresholds);
+  std::chrono::milliseconds getDefaultLatencyThreshold(
```

Comment:
no need for the parameter. just `getDefaultLatencyThreshold()`

Reply:
Ah, good point, thanks!